assets/differentialabundance_report.Rmd

```diff
@@ -467,16 +469,46 @@ print( htmltools::tagList(datatable(contrasts_to_print, caption = paste0("Table
 
 ## Counts
 
-Input was a matrix of `r nrow(assay_data$raw)` `r params$features_type`s for `r ncol(assay_data$raw)` `r params$observations_type`s`r ifelse(nrow(assay_data$normalised) < nrow(assay_data$raw), paste0(', reduced to ', nrow(assay_data$normalised), ' ', params$features_type, 's after filtering for low abundance'), '')`.
+Input was a matrix of `r nrow(assay_data$raw)` `r params$features_type`s for `r ncol(assay_data$raw)` `r params$observations_type`s`r ifelse(nrow(assay_data$normalised) < nrow(assay_data$raw), paste0(', reduced to ', nrow(assay_data$normalised), ' ', params$features_type, 's after filtering for low abundance. The matrix contains '), '')`.
```

Comment:
Hmm, something seems to be missing at the end here; the line ends with "The matrix contains " with no follow-up